app/controllers/application_controller.rb

```diff
@@ -9,11 +9,56 @@ class ApplicationController < ActionController::Base
   before_action -> { cookies.encrypted[:example_id] ||= SecureRandom.uuid }
   before_action :configure_permitted_parameters, if: :devise_controller?
   before_action :set_paper_trail_whodunnit
+  before_action :set_meta_tag_data
 
   impersonates :user
 
   protected
 
+  # See https://github.com/kpumuk/meta-tags#allowed-options-for-display_meta_tags-and-set_meta_tags-methods
+  def set_meta_tag_data
+    set_meta_tags(
+      site: "CodeFund",
+      title: controller_name.to_s.humanize.titleize.gsub("Pages", "Home"),
+      description: "CodeFund is an open ad platform that funds developers through ethical advertising.",
```

Comment:
Should we use the same tag line that's on the readme? Might be good to have some consistency regardless of what the tag line actually is.

> CodeFund Ads is an ethical and discreet ad platform that funds open source.